server/lib/realtime/metrics/prom_ex.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Realtime.Metrics.PromEx do
+  use PromEx, otp_app: :realtime
+
+  @impl true
+  def plugins do
+    [Realtime.Metrics.PromEx.Plugins.Realtime]
```

Comment:
It can be useful for debugging and optimizing adding metrics for VM `Plugins.Beam`

Reply:
sounds good, i'll add it in